Iterators and Closures/Standard Library Types/Intro/task.md

```diff
@@ -232,13 +238,14 @@ Next, we’ll implement the `Iterator` trait for our `Counter` type by defining
 
 We set the associated `Item` type for our iterator to `u32`, meaning the iterator will return `u32` values. Again, don’t worry about associated types yet, we’ll cover them in Chapter 19.
 
-We want our iterator to add 1 to the current state, so we initialized `count` to 0 so it would return 1 first. If the value of `count` is less than 6, `next` will return the current value wrapped in `Some`, but if `count` is 6 or higher, our iterator will return `None`.
+We want our iterator to add 1 to the current state, so we initialized `count` to 0 so it would return 1 first. If the value of `count` is less than 5, `next` will increment `count` and return the current value wrapped in `Some`. Once `count` is 5, our iterator will stop incrementing `count` and always return `None`.
```

Comment:
We want our iterator to add 1 to the current state, so we initialize `count` to 0 so it would return 1 first. If the value of `count` is less than 5, `next` will increment `count` and return the current value wrapped in `Some`. Once `count` is 5, our iterator will stop incrementing `count` and will always return `None`.